SCClassLibrary/Common/Collections/Bag.sc

```diff
@@ -17,15 +17,17 @@ Bag : Collection {
 		});
 	}
 	remove { arg item, count=1;
-		var newCount;
+		var newCount, out;
```

Comment:
why is the variable `out` here? It dosen't seem to be used.

Reply:
My mistake